apps/frontend/src/app/5_pages/StakePage/hooks/useGetStakingBalanceOf.ts

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from 'react';
+
+import { useGetProtocolContract } from '../../../../hooks/useGetContract';
+import { asyncCall } from '../../../../store/rxjs/provider-cache';
+
+export const useGetStakingBalanceOf = (address: string) => {
```

Comment:
Do you need this hook? Can't you just use `useAssetBalance` (it's also cached)?

Reply:
yep, `useAssetBalance` hook shows only your `balance` but we need to show `staked balance`

Reply:
makes sense, thanks for the explanation :)